core/src/data/dataSource.h

```diff
@@ -41,8 +44,8 @@ class DataSource : public std::enable_shared_from_this<DataSource> {
     /* Stops any running I/O tasks pertaining to @_tile */
     virtual void cancelLoadingTile(const TileID& _tile);
 
-    /* Parse a <TileTask> with data into a <TileData>, returning an empty TileData on failure */
-    virtual std::shared_ptr<TileData> parse(const TileTask& _task, const MapProjection& _projection) const = 0;
+    /* FIXME doc /Parse a <TileTask> with data into a <TileData>, returning an empty TileData on failure */
```

Comment:
"Transform the data from a TileTask into Features, which are passed into a TileDataSink" something like that?